classes/install/Upgrade.inc.php

```diff
@@ -1585,17 +1594,17 @@ function convertCommentsToEditor() {
 				
 				$queryDao->insertObject($query);
 				$queryDao->resequence(ASSOC_TYPE_SUBMISSION, $row['article_id']);
-				$queryDao->insertParticipant($query->getId(), $row['user_id']);
+				$queryDao->insertParticipant($query->getId(), $user_id);
 				
 				$queryId = $query->getId();
 
 				$note = $noteDao->newDataObject();
-				$note->setUserId($row['user_id']);
+				$note->setUserId($user_id);
 				$note->setAssocType(ASSOC_TYPE_QUERY);
 				$note->setTitle('Comments to Editor');
```

Comment:
maybe to use here the translated key "submission.submit.coverNote" either in the submission locale or journal primary locale (I am not sure what would be better -- maybe submission locale?)?